src/content/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring.mdx

```diff
@@ -1,325 +1,205 @@
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        Global test coverage
-      </td>
-
-      <td>
-        Check coverage from locations around the world to ensure your users can access your website from anywhere, anytime. Monitor sites behind your firewall by adding the [synthetic monitoring static IP addresses](/docs/apm/new-relic-apm/getting-started/networks#synthetics-public) to your allow list. Use [private locations](/docs/synthetics/new-relic-synthetics/private-locations/private-locations-overview-monitor-internal-sites-add-new-locations) to monitor internal sites or to expand your coverage to new locations.
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        Compatibility with popular analytics platforms
-      </td>
-
-      <td>
-        Synthetic monitoring specifically excludes scripts for [popular analytics services](https://s3.amazonaws.com/nr-synthetics-assets/default-hostnames-blacklist/production/default-hostnames-blacklist.txt), like Google Analytics. This ensures your analytics tools continue to receive the exact same data, even with thousands of monitors checking your website each month.
-
-        You can [unblock](/docs/synthetics/new-relic-synthetics/scripting-monitors/write-scripted-browsers#unblock-service) any of the services blocked by default, or [block](/docs/synthetics/new-relic-synthetics/scripting-monitors/synthetics-scripted-browser-reference#browser-addHostnameToBlacklist) additional services.
-      </td>
-    </tr>
-
-    <tr>
-      <td>
-        REST API functions
-      </td>
-
-      <td>
-        Synthetic monitoring includes a REST API, which you can use to manage:
-
-        * [Simple monitors](/docs/apis/synthetics-rest-api/monitor-examples/manage-synthetics-monitors-via-rest-api) and [scripted monitors](/docs/apis/synthetics-rest-api/monitor-examples/manage-synthetics-monitor-scripts-rest-api)
-        * [Categories and labels for monitors](/docs/apis/synthetics-rest-api/monitor-examples/use-synthetics-label-apis)
-        * [Alert notifications](/docs/apis/synthetics-rest-api/alert-examples/manage-synthetics-alert-notifications-rest-api)
-      </td>
-    </tr>
-  </tbody>
-</table>
+<figcaption>
+Go to **[one.newrelic.com](https://one.newrelic.com) > Add data** > search for **Synthetic monitoring** > choose **Simple browser monitor**. 
+</figcaption>
+</Side>
+</SideBySide>
+
+</Step>
+
+<Step>
+
+## Create a scripted browser monitor [#browser]
+
+Ping monitors won't execute JavaScript because they aren't a full browser, just an HTTP client. Simple browser monitors, while using a Google Chrome instance, don't perform complex checks on your web page. 
+
+Scripted browser monitors give you full Javascript functionality so your monitors can check the availability of the individual elements in your web app. 
```

Comment:
This wording is a little confusing. We may want to indicate that scripted browser monitors allow you to test the availability, functionality, and performance of critical user flows in your web application. We can add that it uses Selenium WebDriver to do so if we want. Calling this full Javascript functionality makes it sound like Javascript isn't processed in simple browser monitors, and that is not the case.